xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1404,19 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last incremental profile stat started and last
+       fop hit.
```

Comment:
I think we should also provide this info for the cumulative stats.

Reply:
done